ftests/core/src/main/java/org/commonjava/indy/ftest/core/infinispan/CacheProducerMergeXmlTest.java

```diff
@@ -53,27 +48,49 @@
     @Before
     public void setup()
     {
-        manager = new DefaultCacheManager(
-                new GlobalConfigurationBuilder().globalJmxStatistics().allowDuplicateDomains( true ).build() );
-        producer = new CacheProducer( new DefaultIndyConfiguration(), manager );
+        System.setProperty( "GlobalJmxStatistics.allowDuplicateDomains", "true" );
```

Comment:
So is this only manageable by system properties in ISPN 9? Or is that the way to affect all ISPN config builders?

Reply:
This can also be added in the infinispan.xml. It is per container, i.e., all caches in a cacheManager. 
This is just for test. When this test is started, Indy has already initiated a CacheProducer bean. But the test wants to create a new CacheProducer instance for test (because it can not get the injected bean). IMO, we can convert it to a junit test rather than a ftest.